config/prow/cluster/ingress-nginx/ingress-nginx-deployment.yaml

```diff
@@ -4,10 +4,10 @@ apiVersion: policy/v1
 kind: PodDisruptionBudget
```

Comment:
This is generated from `config/prow/cluster/ingress-nginx/helm/generate-ingress-nginx-deployments.sh`, correct?

I would suggest to pin the chart version in the script, so that it's reproducible.

Reply:
Yes, it is.
My idea having a script and a fixed yaml description and a script was using the yaml for a reproducible deployment and using the script to update this yaml to the most recent version of the chart.